### DISCUSSION TITLE
Chinese Translations?

### DISCUSSION THREAD

**Discussion by ledisthebest** [Heroic-Games-Launcher/HeroicGamesLauncher#408] [General]
Hi, I like to help translate this project to Chinese(Simplified & Traditional).

But when I visit the Weblate page, it saids " The translation is temporarily closed for contributions due to maintenance, please come back later."

How can I contribute?
Thanks!

**Comment by flavioislima** [comment]
I saw that we had some conflicts, I'll fix it later and let you know. 👍

**Reply by flavioislima** [reply]
@ledisthebest its fixed now :D
